sys/include/bitfield.h

```diff
@@ -89,18 +89,105 @@ static inline bool bf_isset(uint8_t field[], size_t idx)
     return (field[idx / 8] & (1u << (7 - (idx % 8))));
 }
 
+/**
+ * @brief  Perform a bitwise OR operation on two bitfields
+ *         `out = a | b`
+ *
+ * @pre   The size of @p a, @p b and @p out must be at least @p len bits
+ *
+ * @note  This operation will also affect unused bits the bytes that make up
+ *        the bitfield.
```

Comment:
these notes are missing some preposition between `bits` and `the bytes`
```suggestion
 * @note  This operation will also affect unused bits of the bytes that make up
 *        the bitfield.
```